pydantic/type_adapter.py

```diff
@@ -179,41 +183,41 @@ def validate_python(
 
         Args:
             __object: The Python object to validate against the model.
-            strict: Whether to strictly check types. Defaults to None.
-            from_attributes: Whether to extract data from object attributes. Defaults to None.
-            context: Additional context to use during validation. Defaults to None.
+            strict: Whether to strictly check types. Defaults to `None`.
+            from_attributes: Whether to extract data from object attributes. Defaults to `None`.
+            context: Additional context to use during validation. Defaults to `None`.
 
         Returns:
-            T: The validated object.
-
+            The validated object.
         """
         return self.validator.validate_python(__object, strict=strict, from_attributes=from_attributes, context=context)
 
     def validate_json(
         self, __data: str | bytes, *, strict: bool | None = None, context: dict[str, Any] | None = None
     ) -> T:
-        """Validate a JSON string or bytes against the model.
+        """
+        Validate a JSON string or bytes against the model.
 
         Args:
             __data: The JSON data to validate against the model.
-            strict: Whether to strictly check types. Defaults to None.
-            context: Additional context to use during validation. Defaults to None.
+            strict: Whether to strictly check types. Defaults to `None`.
```

Comment:
```suggestion
            strict: Whether to strictly check types. Defaults to `None`.
```
I think it's a bit confusing to describe this as a "whether", then give three choices (`True` / `False` / `None`). I don't know the best way to communicate the meaning to the users, but for context, the way it works (if I understand correctly, anyway) is that `True` means definitely do strict mode, `False` means definitely don't do strict mode, and `None` means use whatever the default is based on the wrapped type / config.

I think it's worth keeping this description short, but it would be nice if we could provide a bit more clarity here. Maybe something like
```suggestion
            strict: Pass a boolean to override the type checking strictness for the wrapped type. Defaults to `None`.
```